lib/resty/etcd/v3.lua

```diff
@@ -200,6 +200,10 @@ function refresh_jwt_token(self)
     -- token exist and not expire
     -- default is 5min, we use 3min
     -- https://github.com/etcd-io/etcd/issues/8287
+    if self.refresh_fail and now() - self.last_auth_time < 60 then
```

Comment:
I don't like the retry hack. The retry should be controlled by the caller.

Reply:
> I don't like the retry hack. The retry should be controlled by the caller.

yeah，but we should control it by the same rule and  we can not export it to the user, the caller is  our self api such as set, delete and others, so we do this logical in every api  is no good then do it in the one method with the same rule.

Reply:
@gy09535
I suspect if this hack fixes the root cause or just hides the real problem. See my comment in the original issue.

Reply:
agree with you.